frontend/package.json

```diff
@@ -3,20 +3,72 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/core": "7.5.5",
```

Comment:
All those dependencies were added automatically after ejecting. Maybe we should separate them from the normal deps?

Reply:
I don't think it's necessary. If you were building it from ground up it would look just like this